app/Http/Controllers/CourseController.php

```diff
@@ -181,7 +188,7 @@ public function associate(Request $request)
 
 		$result = $user->EnrolledIn()->attach($course);
 
-		return ['course' => $course, self::TEACHERS => $course->teachers];
+		return ['course' => $course, 'teachers' => $course->teachers];
```

Comment:
Lo mimso, no creo que valga la pena mandarlo twice

Reply:
@osdagoso siempre si los manda en el course o no?

Reply:
Ah no, this isn't needed. Se puede acceder a la lista de maestros a través de la función de la relación. No lo quise quitar de esta función though porque es del US de @katiearriagam y @rubendelapena y no quería romper esa funcionalidad (igual y todo eso se puede subir en un bug issue a parte, para que quede mejor organizado).

Reply:
Ahorita si no mandan eso al front-end, en la llamada de AJAX sólo recibo los datos del curso.

i.e. no se está mandando dos veces y por eso lo agregamos ahí.

Ésta es la respuesta de AJAX si quitamos lo de teachers. Sólo tiene directamente lo de la tabla `courses`.

![image](https://user-images.githubusercontent.com/14140121/54381490-8dfe6480-4653-11e9-8a1a-4f394d14ad1f.png)

No estoy muy segura de cómo tendría que mandarme el back-end el curso para que tenga acceso a todos sus datos (incluyendo teachers). Please abran un bug si creen que esto necesita un fix.